pkgs/applications/misc/xteddy/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchzip, pkg-config, xorg, imlib2, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "xteddy-${version}";
+  version = "2.2";
+  src = fetchzip {
+    url = "https://deb.debian.org/debian/pool/main/x/xteddy/xteddy_${version}.orig.tar.gz";
+    sha256 = "0sap4fqvs0888ymf5ga10p3n7n5kr35j38kfsfd7nj0xm4hmcma3";
+  };
+  nativeBuildInputs = [ pkg-config makeWrapper ];
+  buildInputs = [ imlib2 xorg.libX11 xorg.libXext ];
+  makeFlags = [ "LIBS=-lXext" ];
+  preBuild = ''
```

Comment:
I would change it to `postPatch` since we are actually patching the sources here